### PR TITLE
Reduce duplication in the `accumulateWheelTicks`/`accumulateTouchTicks` methods (PR 15886 follow-up)

### PR DESCRIPTION
**Pull request by Snuffleupagus**
It just occurred to me that we can reduce some duplication by having the, virtually identical, `accumulateWheelTicks`/`accumulateTouchTicks` methods use a common helper instead that takes advantage of [bracket notation](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics#bracket_notation).

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -2042,32 +2042,23 @@ const PDFViewerApplication = {
...
+    if ((this[prop] > 0 && ticks < 0) || (this[prop] < 0 && ticks > 0)) {
+      this[prop] = 0;
+    }
+    this[prop] += ticks;
+    const wholeTicks = Math.trunc(this[prop]);
+    this[prop] -= wholeTicks;
     return wholeTicks;
   },
 
+  accumulateWheelTicks(ticks) {
```
Comment (calixteman) [inline web/app.js:2056]: Why not inline these two functions ?

Comment (Snuffleupagus) [reply web/app.js]: Is the updated patch what you had in mind?

**Review by calixteman — APPROVED** [review]
LGTM.

File: web/app.js (hunk 1 above)
Comment (calixteman) [reply web/app.js]: Yep exactly.

**Pull request merged**